apps/super-app/src/routes/SupportDetailWip.tsx

```diff
@@ -17,10 +18,24 @@ import './SupportDetailWip.css';
 
 function Header(props: { title: string }) {
   const { title } = props;
+  const navigate = useNavigate();
   return (
     <nav className="support-header">
-      <ArrowBackIosNewIcon />
-      <Typography variant="h4">{title}</Typography>
+      <IconButton
+        aria-label="back"
+        sx={{ color: 'text.primary' }}
```

Comment:
I'm very sure you want `pimary.900`. This rule does nothing, the button is black.

![image](https://user-images.githubusercontent.com/5432896/226129559-f77d15c4-f1be-42c0-9bc9-2f58d2da0a68.png)

See apps/super-app/src/routes/SupportDetailWip.tsx:

```
            <PublicIcon
              sx={{ color: 'primary.900', height: '1.5rem', width: '1.5rem' }}
            />
```

Reply:
Okay, I see the problem, we need to adjust `Theme.ts`.

Reply:
@glencbz ah good eye, lemme check idk why!!